src/btree/bt_page.c

```diff
@@ -319,6 +319,15 @@ __inmem_col_var_repeats(WT_SESSION_IMPL *session, WT_PAGE *page, uint32_t *np)
 static inline bool
 __unstable_skip(WT_SESSION_IMPL *session, const WT_PAGE_HEADER *dsk, WT_CELL_UNPACK *unpack)
 {
+    /*
+     * We should never see a prepared cell, it implies an unclean shutdown followed by a downgrade
+     * (clean shutdown rolls back any prepared cells). Complain and ignore the row.
+     */
+    if (F_ISSET(unpack, WT_CELL_UNPACK_PREPARE)) {
+        __wt_err(session, EINVAL, "unexpected prepared cell found, ignored");
```

Comment:
Could you change this into a `__wt_msg` call? I think we should continue on our merry way if we see this problem.

Reply:
I disagree, this isn't an informational message, it's an error message?

(Unless I'm mistaken, we'll continue on, discarding the row, regardless of whether we use `__wt_msg` or `__wt_err`,  it's just choosing which application handler takes the message.)

Reply:
I agree with @keithbostic.  There's no error return from the function implied here, we'll continue on our merry way as written.